pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -164,10 +167,14 @@ func (n *NodeLabeller) run() error {
 	cpuFeatures := n.getSupportedCpuFeatures()
 	hostCPUModel := n.getHostCpuModel()
 
-	originalNode, err := n.clientset.CoreV1().Nodes().Get(context.Background(), n.host, metav1.GetOptions{})
+	obj, exists, err := n.virtHandlerNodeInformer.GetStore().GetByKey(n.host)
```

Comment:
It seems that the NodeLabeller controller is only triggered by changes to the cluster config. I'm wondering if now it should also trigger on Node changes?

Reply:
Not sure we want to change the logic for those controllers.

Reply:
I think @vladikr has a PR to periodically call it. In the context of his change I would recommend that we ensure that these periodic updates wich get nodes and potentially update nodes have a random delay. It is always surpising how easy it is for a big chung of components to be almost completely in the same state on different nodes, which can lead to load spikes.

Edit: @vladikr plans to tie it to the periodic heartbeat which has random delays already.